extensions/expression-language/ExpressionContextBuilder.h

```diff
@@ -0,0 +1,52 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef EXTENSIONS_EXPRESSION_LANGUAGE_EXPRESSIONCONTEXTBUILDER_H_
+#define EXTENSIONS_EXPRESSION_LANGUAGE_EXPRESSIONCONTEXTBUILDER_H_
+
+#include "core/ProcessContextBuilder.h"
+#include "ProcessContextExpr.h"
+
+namespace org {
+namespace apache {
+namespace nifi {
+namespace minifi {
+namespace core {
+namespace expressions {
+
+/**
+
```

Comment:
need a comment as to why this exists.

Reply:
My gut reaction is: why is this necessary for this PR? This needs to be explained that linking was problematic given the expression language extension was required in some form. This meant that on Windows, where linking is different, we couldn't enable tests without ensuring the pared down EL existed everywhere. To avoid this, a paradigm I've avoided for quite some time was put into place: hot loading the process context through another artifact.